src/components/table.js

```diff
@@ -701,9 +706,10 @@ const SortableHandleDiv = SortableHandle(props => div(props))
  * @param {Object[]} columnSettings - current column list, in order
  * @param {string} columnSettings[].name
  * @param {bool} columnSettings[].visible
+ * @param {bool} style - style override for the button
```

Comment:
Wouldn't this be an object instead of a boolean?

Reply:
thank you!